rllib/connectors/common/add_observations_from_episodes_to_batch.py

```diff
@@ -6,11 +6,10 @@
 from ray.rllib.core.rl_module.rl_module import RLModule
 from ray.rllib.policy.sample_batch import SampleBatch
 from ray.rllib.utils.annotations import override
-from ray.rllib.utils.spaces.space_utils import batch
 from ray.rllib.utils.typing import EpisodeType
 
 
-class AddLastObservationToBatch(ConnectorV2):
+class AddObservationsFromEpisodesToBatch(ConnectorV2):
     """Gets the last observation from a running episode and adds it to the batch.
```

Comment:
Normally, you would only need the last obs in a forward batch (on the env runner side). But yes, since this connector can also be used as a learner connector, I renamed it to "AddObservations..." (not "last" anymore). For the learner side, yes, you do need all observations from the entire episode, not just the last one of course.